frontend/src/components/Comments.jsx

```diff
@@ -0,0 +1,59 @@
+import React , {useState, useRef}from 'react';
+import { Typography, TextField, Button} from '@material-ui/core';
+import { useDispatch } from 'react-redux';
+import { mergeClasses } from '@material-ui/styles';
+import useStyles from './styles';
+import { commentPost } from '../actions/auth';
+import Results from './Results';
+
+const Comments = ({ props }) => {
+  
+   const classes = useStyles ();
+   const [comments, setComments] = useState(props?.comments);
+   const [comment, setComment] = useState('');
+   const user = JSON.parse(localStorage.getItem('profile'));
+   const dispatch= useDispatch();
+   
+   const handleClick = async () =>{
+        const finalComment = `${user.result.name}: ${comment}`
```

Comment:
The variable finalComment is being created, but never being used. Please remove it if it not used.